lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-job.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_AnsibleTower_ConfigurationManager_Job < MiqAeServiceOrchestrationStack
+    expose :job_template, :association => true
```

Comment:
@bzwei 
The create_job which is a class method also needs to be exposed
``
    def self.create_job(template, args = {}) 
      template_object = template.instance_variable_get('@object')
      klass = ManageIQ::Providers::AnsibleTower::ConfigurationManager::Job
      wrap_results(klass.create_job(template_object, args))
    end